vpx_ports/ppc.h

```diff
@@ -0,0 +1,33 @@
+/*
+ *  Copyright (c) 2016 The WebM project authors. All Rights Reserved.
+ *
+ *  Use of this source code is governed by a BSD-style license
+ *  that can be found in the LICENSE file in the root of the source
+ *  tree. An additional intellectual property rights grant can be found
+ *  in the file PATENTS.  All contributing project authors may
+ *  be found in the AUTHORS file in the root of the source tree.
+ */
+
+#ifndef VPX_PORTS_PPC_H_
+#define VPX_PORTS_PPC_H_
+#include <stdlib.h>
+
+#include "vpx_config.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+int ppc_simd_caps(void);
+
+// Earlier gcc compilers doesn't have support for Altivec
+#if !defined(__clang__) && defined(__GNUC__) && __GNUC__ == 4 && \
```

Comment:
VSX might require a newer version. I'd use 6 to be safe, 5.4 is known to have horrible bugs =P